bundles/org.openhab.binding.meteoalerte/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -9,33 +9,15 @@
 		<description>Fournit les niveaux d'alerte météo pour un département.</description>
 
 		<channels>
-			<channel id="vague-submersion" typeId="alert-level">
-				<label>Etat Vague Submersion</label>
-			</channel>
-			<channel id="vent" typeId="alert-level">
-				<label>Etat Vent</label>
-			</channel>
-			<channel id="pluie-inondation" typeId="alert-level">
-				<label>Etat Pluie Inondation</label>
-			</channel>
-			<channel id="orage" typeId="alert-level">
-				<label>Etat Orage</label>
-			</channel>
-			<channel id="inondation" typeId="alert-level">
-				<label>Etat Inondation</label>
-			</channel>
-			<channel id="neige" typeId="alert-level">
-				<label>Etat Neige</label>
-			</channel>
-			<channel id="canicule" typeId="alert-level">
-				<label>Etat Canicule</label>
-			</channel>
-			<channel id="grand-froid" typeId="alert-level">
-				<label>Etat Grand Froid</label>
-			</channel>
-			<channel id="avalanches" typeId="alert-level">
-				<label>Etat Avalanches</label>
-			</channel>
+			<channel id="vague-submersion" typeId="vague-submersion"/>
```

Comment:
Migration instructions would be appreciated as you changed all thing types.

Reply:
Done

Reply:
I neutralized the update instructions currently because I'm facing the same error than reported [here](https://github.com/openhab/openhab-distro/issues/1519).